arangod/RestHandler/RestCollectionHandler.cpp

```diff
@@ -159,8 +156,13 @@ RestStatus RestCollectionHandler::handleCommandGet() {
       {
         VPackObjectBuilder obj(&_builder, true);
 
-        obj->add("checksum", VPackValue(std::to_string(checksum)));
-        obj->add("revision", VPackValue(revId.toString()));
+        if (raw) {
+          obj->add("checksum", VPackValue(checksum));
+          obj->add("revision", VPackValue(revId.id()));
+        } else {
+          obj->add("checksum", VPackValue(std::to_string(checksum)));
+          obj->add("revision", VPackValue(revId.toString()));
+        }
```

Comment:
I would be in favor of not making this distinction, but keep the APIs as simple as possible.
This would mean we will need to send the "checksum" and "revision" values back as strings, as before. This should be fine, because all clients are expecting it like that already.
This only makes parsing of the individual results from DB servers slightly more complicated, because now a string needs to be turned into a number before comparing/xoring.
But it should be doable easily, and efficiency should not matter, as we are talking only about two attributes in each response.